pom.xml

```diff
@@ -27,19 +27,19 @@
         <skip.integration.tests>false</skip.integration.tests>
 
         <!-- General Selenium run versions -->
-        <selenium.version>3.8.0</selenium.version>
-        <webdrivermanager.version>2.1.0</webdrivermanager.version>
+        <selenium.version>3.12.0</selenium.version>
+        <webdrivermanager.version>2.2.1</webdrivermanager.version>
         <phantomjsdriver.version>2.0.0</phantomjsdriver.version>
-        <slf4j.version>1.7.23</slf4j.version>
-        <testng.version>6.9.9</testng.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <testng.version>6.14.3</testng.version>
 
         <!-- Additional plugin versions -->
-        <maven.compiler.plugin.version>2.3.2</maven.compiler.plugin.version>
+        <maven.compiler.plugin.version>2.7.0</maven.compiler.plugin.version>
```

Comment:
this version doesn't appear to exist? Were you looking for 3.7.0?
https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-compiler-plugin

Reply:
yes sorry